src/contracts/core/DelegationManager.sol

```diff
@@ -37,9 +37,16 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         _;
     }
 
+    // @notice Simple permission for functions that are only callable by the StrategyManager contract OR by the EigenPodManagerContract
+    modifier onlyStrategyManagerOrEigenPodManager() {
```

Comment:
Would prefer this being a mapping. I can see us wanting permissionless strategy managers that can push updates to the delegation manager for the strategies that they "own".

Reply:
> Would prefer this being a mapping. I can see us wanting permissionless strategy managers that can push updates to the delegation manager for the strategies that they "own".

This would break (rather than just modifying) the invariant we have w/r/t delegation, that someone can only undelegate if they have no delegated shares. for now, this is cheaper than a mapping since both addresses are stored immutably

Reply:
We can make the change when we need to, I guess